database.json

```diff
@@ -3079,6 +3079,12 @@
     "repo": "deno-watch",
     "desc": "A pure deno file watcher."
   },
+  "web-name": {
+    "type": "github",
+    "owner": "letskhabar",
+    "repo": "web-name",
+    "desc": "simple module to generate random usernames"
+  },
```

Comment:
database names with dashes are not allowed so can change the name to web_name, webName something else

Reply:
Thanks for suggestions 😊😘

Reply:
> database names with dashes are not allowed so can change the name to web_name, webName something else

@alirealasad Are module names case-sensitive? That would be a disaster having different modules named `webname`, `webName`, `Webname`, `wEbname` and so on

Reply:
Web_name is also a option